hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -48,6 +49,7 @@
   private Timer deltaCommitTimer = null;
   private Timer finalizeTimer = null;
   private Timer compactionTimer = null;
+  private Timer clusteringTimer = null;
```

Comment:
This is specific to clustering operation and only used in WriteClient#cluster. 
We could have a different timer for insertOverwrite and replace commands if needed. I like this approach because replace can mean multiple things. Let me know if you think common timer makes more sense.